packages/accordions/src/elements/accordion/Accordion.tsx

```diff
@@ -0,0 +1,119 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, {
+  useRef,
+  useEffect,
+  forwardRef,
+  RefAttributes,
+  HTMLAttributes,
+  PropsWithoutRef,
+  ForwardRefExoticComponent
+} from 'react';
+import { useAccordion } from '@zendeskgarden/container-accordion';
+import { StyledAccordion } from '../../styled';
+import { AccordionContext } from '../../utils';
+import { Section } from '../accordion/components/Section';
+import { Header } from '../accordion/components/Header';
+import { Label } from '../accordion/components/Label';
+import { Panel } from '../accordion/components/Panel';
+
+interface IStaticAccordionExport<T, P>
+  extends ForwardRefExoticComponent<PropsWithoutRef<P> & RefAttributes<T>> {
+  Section: typeof Section;
+  Header: typeof Header;
+  Label: typeof Label;
+  Panel: typeof Panel;
+}
+
+interface IAccordionProps extends Omit<HTMLAttributes<HTMLDivElement>, 'onChange'> {
+  /** The `aria-level` used to apply heading rank on accordion headers */
+  level: number;
+  /** A controlled prop that determines which sections are expanded */
```

Comment:
> a controlled prop

Do we need to specify that this is "controlled"? I'm used to the pattern of just describing what the prop is related to. i.e. "Determines which sections are expanded"

Reply:
Probably not, I'll update the prop description to "Determines which sections are expanded".